### PR TITLE
Commercial invoice simplification

### PR DESCRIPTION
**Pull request by nissimsan**
This inlines and simplifies and tightens up Commercial Invoice:
- inlining
- simplify portOfEntry
- optional consignee
- remove postalBox
- itemsShipped minItems=1
- remove name, descr, related links
- country code type
- req id

### CODE REVIEW

**Review by BenjaminMoe — APPROVED** [review]

File: docs/openapi/components/schemas/common/Invoice.yml [hunk 1]
```diff
@@ -396,12 +395,12 @@ example: |-
         }
       }
     },
-    "consignee": {
+    "consignee": [{
```
Comment (OR13) [inline docs/openapi/components/schemas/common/Invoice.yml:398]: is this really a "multiple". field?

**Review by OR13 — APPROVED** [review]

File: docs/openapi/components/schemas/common/Invoice.yml (hunk 1 above)
Comment (mprorock) [inline docs/openapi/components/schemas/common/Invoice.yml:398]: I suppose if we permit intermediates, but will need to signify ultimately who is final

**Review by mprorock — APPROVED** [review]
ok with this, but would prefer either a single consignee, or a way of identifying intermediaries

**Review by mkhraisha — APPROVED** [review]
I think single consignee maps cleaner but we don't currently have a requirement for it.

**Review by mkhraisha — APPROVED** [review]

File: docs/openapi/components/schemas/credentials/CommercialInvoiceCredential.yml [hunk 2]
```diff
@@ -72,19 +167,1153 @@ properties:
...
+                  required:
+                    - type
+              additionalProperties: false
+              required:
+                - type
+          additionalProperties: false
+          required:
+            - type
+        minItems: 0
+        maxItems: 1
```
Comment (BenjaminMoe) [inline docs/openapi/components/schemas/credentials/CommercialInvoiceCredential.yml:584]: Here

**Pull request merged**